### PR TITLE
ISSUE_TEMPLATE changes

### LINKED ISSUE #61094: [triage] remove "I want help writing my application" from "New Issue" options
## Use case

Opening a new issue the reporter finds in the [landing page](https://github.com/flutter/flutter/issues/new/choose) the option [I want help writing my application](https://user-images.githubusercontent.com/45637986/86916953-07fe0b00-c12d-11ea-8fd5-0ac5cc6964f5.png);
I believe, for this specific type of issues, the reporters could receive better support 
in the platforms indicated in the [community page](https://flutter.dev/community), on [Stack Overflow ](https://stackoverflow.com/questions/tagged/flutter) and in the [flutterhelp](https://www.reddit.com/r/flutterhelp/ ) sub-reddit.

## Proposal

remove from the [new issue options](https://github.com/flutter/flutter/issues/new/choose) `I want help writing my application`
providing instead a link to the [community page](https://flutter.dev/community) and [flutterhelp](https://www.reddit.com/r/flutterhelp/ ) sub-reddit.

Thank you

### PR DESCRIPTION
**Pull request by iapicca**
## Description

Inspired by [vs code approach ](https://github.com/microsoft/vscode/issues/new/choose) and [implementation](ISSUE_TEMPLATE) following github's [documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) the file `config.yml`
as been added to `.github/ISSUE_TEMPLATE` and `SUPPORT.md`has been removed for the same.

This would affect the flow of opening `new issues`; a user won't be able to open a new issue with the selection 
`I want help writing my application`, but it would be redirected to the platform of her or his choice between
 https://www.reddit.com/r/flutterhelp/ and  https://stackoverflow.com/tags/flutter

## Related Issues

Fixes https://github.com/flutter/flutter/issues/61094

## Tests

I added the following tests:

no test added

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ x] I signed the [CLA].
- [ x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ x] I updated/added relevant documentation (doc comments with `///`).
- [ x] All existing and new tests are passing.
- [ x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [ x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x ] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: https://flutter.dev/go/breaking-changes-template *Replace this with a link to a pull request that adds the migration guide to https://flutter.dev/docs/release/breaking-changes*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: I want help writing my application from Flutter's community on Reddit
```
Comment (goderbauer) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Can we only add the link to stackoverflow? Too many choices may make it difficult to to chose one.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    url: https://www.reddit.com/r/flutterhelp/
+    about: Get help from the Flutter's community on Reddit to achieve a particular effect, or with using a particular API.
```
Comment (goderbauer) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Maybe: Ask your questions about how to achieve a particular effect in Flutter or get help with using a particular API.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (iapicca) [reply .github/ISSUE_TEMPLATE/config.yml:3]: sounds good to me, I've included both Reddit and SO following this [comment](https://github.com/flutter/flutter/issues/61094#issuecomment-668777210)

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (iapicca) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Sure thing

**Comment by iapicca** [comment]
@goderbauer 
I've removed the Reddit's `contact_links` and changed `name` and `about` for the SO one

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**